README.md

```diff
@@ -1 +1,84 @@
-# s-covid
+# SCovid
+
+
+
+## Introduction
+
+**SCovid** is a NodeMCU (ESP8266 development board) based sensor for CO2 monitoring. 
+
+It has been developed mainly for teachers (educational environments), in order to facilitate the measures of CO2 level at school.
+
+![enter image description here](Sentsoreak.png "Sensors")
+
+It measures the following magnitudes: 
+
+	 - CO2 level    pppm 
```

Comment:
- CO2 level is measured in "ppm" (parts per million)

Reply:
Done